components/ipfs/ipfs_p3a.cc

```diff
@@ -46,44 +46,86 @@ int GetDaemonUsageBucket(base::TimeDelta elapsed_time) {
 void RecordIPFSDetectionPromptCount(PrefService* prefs) {
   const int max_bucket = 4;
   int bucket = GetIPFSDetectionPromptBucket(prefs);
-  UMA_HISTOGRAM_EXACT_LINEAR("Brave.IPFS.DetectionPromptCount", bucket,
+  UMA_HISTOGRAM_EXACT_LINEAR(kDetectionPromptCountHistogramName, bucket,
                              max_bucket);
 }
 
 // IPFS state
 // Ask (0), Gateway (1), Local Node (2), Disabled (3)
 void RecordIPFSGatewaySetting(PrefService* prefs) {
   auto resolve_method = prefs->GetInteger(kIPFSResolveMethod);
-  UMA_HISTOGRAM_EXACT_LINEAR("Brave.IPFS.GatewaySetting", resolve_method, 4);
+  UMA_HISTOGRAM_EXACT_LINEAR(kGatewaySettingHistogramName, resolve_method, 4);
+}
+
+// Was the IPFS local node installed? If so, is it still used?
+// Never installed (0), Installed (1), Installed, not used any more (2)
```

Comment:
personally I'd rather add a link to the wiki or use in place comments, because people tend to forget fixing comments while changing the code